### DISCUSSION TITLE
meld: cannot compare files in ~/.config/firejail

### DISCUSSION THREAD

**Discussion by githlp** [netblue30/firejail#4997] [Q&A] [answered]
There was a problem opening the file “/home/ronald/firefox.local”.

But my actual working dir where i called meld is "/home/ronald/.config/firejail" ..  confused..

thats is my actual meld.local:
`whitelist /tmp/dbus-NbthRTyzDW`
`noblacklist ${HOME}/.config/firejail`

**Comment by rusty-snake** [comment, marked as answer]
>  noblacklist ${HOME}/.config/firejail

You can not `noblacklist` `${HOME}/.config/firejail`, if you want to compare files in ` ${HOME}/.config/firejail` you need to `/usr/bin/meld`.

> But my actual working dir where i called meld is "/home/ronald/.config/firejail" .. confused..

As said above, `${HOME}/.config/firejail` is always blacklisted. If firejail can not cd into the (implicit) requested cwd, it cds into `~` (and if that fails too `/`) AFAICTY.

**Comment by githlp** [comment]
oha, i forgot about that!

Your answers are just lightening-fast, thx again!
